spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/interpreter/DynamicInterpreter.scala

```diff
@@ -65,6 +65,13 @@ object DynamicInterpreter {
     conformedDf
   }
 
+  def findRulesBreakingImmutability(steps: List[ConformanceRule], schema: StructType): Seq[DropConformanceRule] = {
```

Comment:
```suggestion
  def findOriginalColumnsDropRules(steps: List[ConformanceRule], schema: StructType): Seq[DropConformanceRule] = {
```

Reply:
the method name should probably follow the suit of the final configuration field name, (_original_ should be included, _drop_ maybe not). `findOriginalColumnsModificationRules`?

Reply:
Made it private. Used naming from previous comment